app/core/Router.js

```diff
@@ -36,11 +36,12 @@ const ViewLoadTimer = require('core/ViewLoadTimer')
 const paymentUtils = require('lib/paymentUtils')
 
 const homePageExperiment = function () {
```

Comment:
could get rid of this function and use one in user as `me`.

Also, not a bad idea to change experiment name even though we can query by date